### DISCUSSION TITLE
Sending email to many recipients

### DISCUSSION THREAD

**Discussion by gpbuil** [adonisjs/core#1967] [General]
Hi There,

I am trying to send a message using adonisjs mail, but need to add to message the .cc() recipients based on an array.
const users_cc = ['email1@emai.com', 'email2@email.com']
			await Mail.send(
				'emails.evento',
				{
					user: user[0].nome,
					data: data,
				},
				(message) => {
					message
						.from('from@email.com')
						.to(users)
                                                .cc(users_cc)
						.subject('Cadastro em evento')
				}
			)

How could I loop into users_cc adding message.cc(users_cc[0]), message.cc(users_cc[1]), and so on?

Many thanks,

Gines

**Comment by thetutlage** [comment]
Follow the [Github markdown](https://guides.github.com/features/mastering-markdown/) guide and please format your code blocks properly.

**Comment by mathieuh** [comment]
Hi,

Like said before you should format your code to be more readable please.

Looking at mail code you can give an array to cc like this: `message.cc([{ address: 'foo@bar.com', name: 'Foo' }])`
